testing/src/test/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -92,7 +92,9 @@ public void setup(StargateConnectionInfo cluster) throws IOException {
 
   @AfterEach
   public void teardown() {
-    session.execute(String.format("drop keyspace %s", keyspace));
+    if (Math.random() < 0.5) {
+      session.execute(String.format("drop keyspace %s", keyspace));
+    }
```

Comment:
Do we intend to merge this?

Reply:
No - I'm just messing around for now

Reply:
Ah... mind converting the PR to "Draft"? :-)

Reply:
Will do sorry :P